src/pages/Organization/components/OrganizationSelector.tsx

```
@@ -110,16 +110,13 @@ export default function OrganizationSelector(props: OrganizationSelectorProps) {
     <>
       {/* Selected Levels */}
       {selectedLevels.map((level, index) => (
-        <div key={index}>
+        <div>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)